src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline

```diff
@@ -1353,6 +1353,13 @@ google.auth.getApplicationDefault(function(err, authClient) {
       // TODO: Add desired properties of the request body.
     },
 
+    // TODO: Use `alt: 'media'` to download media content.
```

Comment:
For Node.js, there is not a corresponding doc for media download that I could find. IIRC, the closest thing is instructions specific to the Google Drive API. The included URL above is specific to CORS, and does not address media download per se.